css/styles.css

```diff
@@ -18,45 +18,54 @@
     height: 100px;
     width: 98%;
     background-color: blue;
-    margin-top: 50px;
+    /* margin-top: 50px;
     padding-top: 10px;
-    border: 1px solid black;
+    border: 1px solid black; */
 }
 
 .nav{
     height: 50px;
     width: 98%;
     background-color:green;
     /* margin-top: 50px; */
-    padding-top: 10px;
+    /* padding-top: 10px; */
     border: .5px solid black;
-    padding-left: 10px;
+    /* padding-left: 10px; */
 }
 
 .row{
     width: 100%;
```

Comment:
there is no need to manipulate `width` due to the `row` on its own does that through Bootstrap.